### PR TITLE
Pal 1082 remove resource parent

### PR DESCRIPTION
**Pull request by dev930018**
Awaiting on [PAL-1023](https://github.com/gchq/Palisade-services/pull/332) being merged

### CODE REVIEW

File: policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/config/PolicyConfiguration.java [hunk 1]
```diff
@@ -29,7 +29,8 @@
      *
      * @return a {@link List} of the objects that have implemented {@link PolicyPrepopulationFactory}.
      */
-    @SuppressWarnings("java:S1452") // Allow generics wildcard
+    @SuppressWarnings("java:S1452")
+    // Allow generics wildcard
```
Comment (pd104923) [inline policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/config/PolicyConfiguration.java]:
```suggestion
    // Allow generics wildcard
    @SuppressWarnings("java:S1452")
```
Should the comment come before the suppression?

File: topic-offset-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/topicoffset/kafka/KafkaContractTest.java [hunk 2]
```diff
@@ -91,7 +91,8 @@
 )
 @Import(KafkaTestConfiguration.class)
 @ActiveProfiles({"akka-test", "testcontainers"})
-@SuppressWarnings("java:S1774") //Suppress ternary operator smell
+@SuppressWarnings("java:S1774")
+        //Suppress ternary operator smell
```
Comment (pd104923) [inline topic-offset-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/topicoffset/kafka/KafkaContractTest.java]:
```suggestion
// Suppress ternary operator smell
@SuppressWarnings("java:S1774")
```

File: topic-offset-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/topicoffset/kafka/KafkaTestConfiguration.java [hunk 3]
```diff
@@ -83,7 +83,8 @@ static PropertiesConfigurer propertiesConfigurer(final ResourceLoader resourceLo
     }
 
     @Bean
-    @SuppressWarnings("java:S112") // Suppress specific exception smell
+    @SuppressWarnings("java:S112")
+        // Suppress specific exception smell
```
Comment (pd104923) [inline topic-offset-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/topicoffset/kafka/KafkaTestConfiguration.java]:
```suggestion
    // Suppress specific exception smell
    @SuppressWarnings("java:S112")
```
I think the comment should describe the smell that's being suppressed

**Review by pd104923 — CHANGES_REQUESTED** [review]
Some small changes

File: NOTICES.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ List of third-party dependencies grouped by their license type
 * spring-boot-starter-web ([org.springframework.boot:spring-boot-starter-web:2.3.1.RELEASE](https://spring.io/projects/spring-boot))
 * Spring Cloud Starter OpenFeign ([org.springframework.cloud:spring-cloud-starter-openfeign:2.2.2.RELEASE](https://projects.spring.io/spring-cloud))
 * event-logging ([uk.gov.gchq.eventlogging:event-logging:4.0.8_schema-v3.2.4](https://github.com/gchq/event-logging))
-* GCHQ Palisade Common Library ([uk.gov.gchq.palisade:common:0.5.0-RELEASE](https://github.com/gchq/Palisade-common))
```
Comment (developer01189998819991197253) [inline NOTICES.md]: revert

File: attribute-masking-service/mvn_dependency_tree.txt [hunk 5]
```diff
@@ -1,5 +1,5 @@
-uk.gov.gchq.palisade:attribute-masking-service:jar:0.5.0-RELEASE
```
Comment (developer01189998819991197253) [inline attribute-masking-service/mvn_dependency_tree.txt]: revert, we're currently at a RELEASE

File: attribute-masking-service/src/main/java/uk/gov/gchq/palisade/service/attributemask/model/AttributeMaskingRequest.java [hunk 6]
```diff
@@ -224,7 +224,7 @@ default IResource withUser(User user) {
             /**
              * Adds the rules that specify the access.
              *
-             * @param <T> the domain class for the rules
+             * @param <T>   the domain class for the rules
```
Comment (developer01189998819991197253) [inline attribute-masking-service/src/main/java/uk/gov/gchq/palisade/service/attributemask/model/AttributeMaskingRequest.java:227]:
```suggestion
             * @param <T> the domain class for the rules
```

File: audit-service/mvn_dependency_tree.txt [hunk 7]
```diff
@@ -1,5 +1,5 @@
-uk.gov.gchq.palisade:audit-service:jar:0.5.0-RELEASE
-+- uk.gov.gchq.palisade:common:jar:0.5.0-RELEASE:compile
+uk.gov.gchq.palisade:audit-service:jar:0.5.0-SNAPSHOT
```
Comment (developer01189998819991197253) [inline audit-service/mvn_dependency_tree.txt]: revert changes to dependency trees

File: data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/service/DataSerialiserPrePopTest.java [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+import uk.gov.gchq.palisade.service.data.config.StdSerialiserPrepopulationFactory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assumptions.assumeThat;
+
+@SpringBootTest
+@ContextConfiguration(classes = {SerialiserConfiguration.class})
+@ActiveProfiles({"test-serialisers"})
+// Fails to consistently work for unknown reasons
+@Disabled
```
Comment (developer01189998819991197253) [inline data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/service/DataSerialiserPrePopTest.java]: delete disabled test

File: services-manager/README.md [hunk 9]
```diff
@@ -15,7 +15,7 @@ limitations under the License.
 --->
 
 <!---
-The contents of this file are under substitution in the build process - maven's `process-resources` stage will substitute executable(dot)jar for this: services-manager-0.5.0-RELEASE-exec.jar
+The contents of this file are under substitution in the build process - maven's `process-resources` stage will substitute executable(dot)jar for this: services-manager-0.5.0-SNAPSHOT-exec.jar
```
Comment (developer01189998819991197253) [inline services-manager/README.md]: revert changes

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]

File: attribute-masking-service/src/main/java/uk/gov/gchq/palisade/service/attributemask/model/AttributeMaskingRequest.java (hunk 6 above)
Comment (dev930018) [reply attribute-masking-service/src/main/java/uk/gov/gchq/palisade/service/attributemask/model/AttributeMaskingRequest.java:227]: Indent to match javadoc line below

**Review by developer01189998819991197253 — APPROVED** [review]

**Pull request merged**